### DISCUSSION TITLE
Additions to /invsee

### DISCUSSION THREAD

**Discussion by 4zyy** [EssentialsX/Essentials#5930] [Ideas and Feature Suggestions]
1) Can we please see the players offhand as well as their armour
2) Can /invsee work for offline players too
3) Can /endsee also work for offline players

**Comment by Evidentsinger14** [comment]
1. You can view a players armor slots by adding an argument after the player `/invsee <player> armor` (non-modifyable)
2. No, this has been previously denied, and will continue to be. Use a dedicated inventory management plugin for that
3. `/endsee` is not an EssentialsX command, you can view, and modify the enderchest of another online player with `/enderchest <player>`, also has been denied for offline player support, see number 2.
